build/scripts/tests.sh

```diff
@@ -14,7 +14,7 @@ root_path="$(get_repo_dir)"
 binaries_path="${root_path}"/Binaries
 unittest_dir="${binaries_path}"/"${build_configuration}"/UnitTests
 log_dir="${binaries_path}"/"${build_configuration}"/xUnitResults
-nuget_dir="${HOME}"/.nuget/packages
+nuget_dir="${HOME}"/.nuget/roslyn-packages
```

Comment:
> it's important to keep the case where NUGET_PACKAGES is unset working, as the current advice is to call build/scripts/tests.sh manually if you want to test on linux, and presumably the user doesn't have NUGET_PACKAGES set

We can't hardcode this here, because we tell users to run this script manually (after restoring with an unset NUGET_PACKAGES). See my comment for an example of a correct fix.

Reply:
Need to add a shell helper equivalent of the PS Get-PackagesDir function